<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaSeq</class>
 <widget class="QDialog" name="ListaSeq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #c5ccb8;
color: #10121c;</string>
  </property>
  <widget class="QLabel" name="label_lista">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>301</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #9a9a97;
border:  2px solid #6f6776;
border-radius: 20px;
color: #10121c;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000000;</string>
   </property>
   <property name="text">
    <string>Lista Sequencial:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;</string>
   </property>
   <property name="text">
    <string>by: Virgilio Schettini de Oliveira Neto - 20200013848</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>347</width>
     <height>308</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>MS Sans Serif</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000000;</string>
          </property>
          <property name="text">
           <string>Valor a ser inserido:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_add_valor">
          <property name="styleSheet">
           <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
background: #ffffff;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>MS Sans Serif</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000000;</string>
          </property>
          <property name="text">
           <string>Posição na lista:      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_add_pos">
          <property name="styleSheet">
           <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
background: #ffffff;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_lista">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 5px;
outline: none;
color: #000000;
background: #9a9a97;</string>
        </property>
        <property name="text">
         <string>Adicionar na lista</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>MS Sans Serif</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000000;</string>
          </property>
          <property name="text">
           <string>Posição da lista para remover:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_rem_pos">
          <property name="styleSheet">
           <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
background: #ffffff;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_rem_lista">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 5px;
outline: none;
color: #000000;
background: #9a9a97;</string>
        </property>
        <property name="text">
         <string>Remover na lista</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>MS Sans Serif</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000000;</string>
          </property>
          <property name="text">
           <string>Posição da lista para consultar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_consulta">
          <property name="styleSheet">
           <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
background: #ffffff;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_consulta">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #6f6776;
border-radius: 5px;
padding: 5px;
outline: none;
color: #000000;
background: #9a9a97;</string>
        </property>
        <property name="text">
         <string>Consultar na lista</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>